scipy/optimize/_differentialevolution.py

```diff
@@ -45,11 +45,16 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         to ``len(x)``.
     bounds : sequence or `Bounds`
         Bounds for variables. There are two ways to specify the bounds:
-        1. Instance of `Bounds` class.
-        2. ``(min, max)`` pairs for each element in ``x``, defining the finite
-        lower and upper bounds for the optimizing argument of `func`.
+
+            1. Instance of `Bounds` class.
+            2. ``(min, max)`` pairs for each element in ``x``, defining the
+               finite lower and upper bounds for the optimizing argument of
+               `func`.
+
         The total number of bounds is used to determine the number of
-        parameters, N.
+        parameters, N. If there are parameters whose bounds are equal the total
+        number of free parameters is ``N - N_equal``.
```

Comment:
I'd suggest defining `N_equal` next time it's convenient.

And maybe add the comma after "whole bounds are equal". (I avoid getting hung up on minor grammar remarks, but it would improve the readability here.)

Reply:
I can add the comma. From the context I thought that it would obvious what `N_equal` would represent, perhaps not.

Reply:
It probably is. I just try to be explicit as a rule because there are many instances in our documentation where symbols aren't defined and it really isn't obvious. I don't want to add to that.